CodingTracker.AshtonLeeSeloka/Services/DataService.cs

```diff
@@ -0,0 +1,133 @@
+﻿using CodingTracker.AshtonLeeSeloka.Models;
+using Dapper;
+using Microsoft.Data.Sqlite;
+using System.Configuration;
+
+
+namespace Services
+{
+	public class DataService
+	{
+		private string? _DBConnectionString = ConfigurationManager.AppSettings.Get("DBConnectionString");
+		private List<CodingSession> _Sessions = new List<CodingSession>();
+		private CalculationsService _CalculationsService = new CalculationsService();
+		private ValidationService _Validation = new ValidationService();
+
+
+		/// <summary>
+		/// Creates a new Instance of DB Coding_Sessions if not Present
+		/// </summary>
+		public void CreateDB()
+		{
+			try
+			{
+				using (var connection = new SqliteConnection(_DBConnectionString))
+				{
+					connection.Open();
+					var tableCmd = connection.CreateCommand();
+					tableCmd.CommandText = @"CREATE TABLE IF NOT EXISTS Coding_Sessions(
+																Id INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
Three unnecessary empty lines! Why? 😄

Reply:
Lol Apologies, will be more mindful in future,
I owe you a coffee one day for the patience lol.

I've resubmitted for your review.